tests/phpunit/includes/Meta/Test_Metadata.php

```diff
@@ -56,57 +57,55 @@ public static function create_pods() {
 
 		foreach ( self::$pod_names as $type => $name ) {
 
-			self::$pod_ids[ $type ] = pods_api()->save_pod(
+			pods_api()->save_pod(
```

Comment:
Saving and passing the pod id is faster for testing purposes, it's not necessary but if we're looking for tests to run quickly and not time out -- it's helpful